### PR TITLE
[Tool] support numa in startup script

### PR DESCRIPTION
**Pull request by murphyatwork**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [x] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Support NUMA binding in the startup script:
```bash
# Bind process to NUMA Node 0
be/bin/start_be.sh --daemon --numa 0 

# Bind process to NUMA Node 1
be/bin/start_be.sh --daemon --numa 1

# Bind to invalid NUMA Node would get error
be/bin/start_be.sh --daemon --numa 1024
```

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by mofeiatwork** [comment]
run starrocks_admit_test

File: bin/start_backend.sh [hunk 1]
```diff
@@ -108,6 +111,15 @@ if [[ "$JAVA_VERSION" -gt 8 ]]; then
     fi
 fi
 
+# check NUMA setting
+NUMA_CMD=""
+if [[ "${RUN_NUMA}" -ne "-1" ]]; then
+    set -e
+    NUMA_CMD="numactl --cpubind ${RUN_NUMA} --membind ${RUN_NUMA}"
+    ${NUMA_CMD} echo "Running on NUMA ${RUN_NUMA}"
```
Comment (silverbullet233) [inline bin/start_backend.sh:119]: if this numa node is not existed, we should exit here.

**Review by satanson — APPROVED** [review]

**Review by ZiheLiu — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: bin/start_backend.sh (hunk 1 above)
Comment (mofeiatwork) [reply bin/start_backend.sh:119]: `set -e` would make the script exit if numa node not exists

**Comment by mofeiatwork** [comment]
@Mergifyio backport branch-2.5 branch-3.0

**Pull request merged**